ui/src/shared/apis/index.ts

```diff
@@ -237,5 +244,5 @@ export const getQueryConfigAndStatus = (url, queries, tempVars) =>
   AJAX({
     url,
     method: 'POST',
-    data: {queries, tempVars},
+    data: { queries, tempVars },
```

Comment:
here's an example of where `make lint` would impose our linter formatting, as mentioned above